src/directives/hover/README.md

```diff
@@ -50,7 +50,7 @@ The directive has no modifiers.
 
 ## Live example
 
-In the following, we are swapping icons and tet color depending on the hover state of the element:
```

Comment:
@tmorehouse This is the first time I see this form of typo for `the`.
Good to know - so I can add it to the list of things I have to look for 😅

Reply:
Oops that should be `text color` and not `the color` (bad `x` key on the keyboard)

Reply:
I have fixed it.